projects/media-viewer/src/lib/viewers/pdf-viewer/annotation-set.service.ts

```diff
@@ -27,27 +27,6 @@ export class AnnotationSetService {
     this.annotationSet = annotationSet;
   }
```

Comment:
seems like the whole service needs to go. I don't think any of it is used

Reply:
yeah good spot